packages/data-models/src/lexeme.js

```diff
@@ -50,6 +50,40 @@ class Lexeme {
     this.addInflections(inflections)
     this.meaning = meaning || new DefinitionSet(this.lemma.word, this.lemma.languageID)
     this.disambiguated = false
+    this.selectedInflection = null
+  }
+
+  /**
+   * Set the selected inflection for a lexeme which has had its
+   * inflections disambiguated
+   * @param {Inflection} inflection the selected inflection
+   */
+  setSelectedInflection (inflection) {
+    this.selectedInflection = inflection
+  }
+
+  /**
+   * Get the selected inflection for a lexeme which has had its
+   * inflections disambiguated
+   * @returns {Inflection} (or null if none is selected)
+   */
+  getSelectedInflection () {
+    return this.selectedInflection
+  }
+
+  /**
+   * Gets the selected inflection formatted for display
+   * (returns an array because the display is grouped by feature
+   * but there should only be one inflection in the array)
+   * @returns {Array} if no selected inflection the array will be empty
+   */
+  getSelectedInflectionForDisplay () {
+    if (this.selectedInflection) {
+      const lm = LMF.getLanguageModel(this.lemma.languageID)
+      return lm.groupInflectionsForDisplay([this.selectedInflection])
```

Comment:
This method is deprecated, I think it's better to use the static variant of it. We won't also need to get the language model for it.

Reply:
ah, okay thanks for that catch!

Reply:
I think I still need the language model though, because the method itself is language model class specific (e.g. syriac language model implements this differently than the base language model class)

Reply:
Then, I guess, `lm.constructor.groupInflectionsForDisplay()` should do

Reply:
it doesn't work for some reason. Possibly because we don't do the inheritance right. As I really hope this method will go away altogether and be replaced by something better with the lexical query refactoring I'm going to leave it using the deprecated method for now and make sure we have an issue to clean this all up.